src/main/java/com/mercadopago/resources/order/OrderRefund.java

```diff
@@ -23,6 +23,11 @@ public class OrderRefund extends MPResource {
     /** Amount Refund. */
     private String amount;
 
-    /** Status Refund. */
+    /** Refund status. */
     private String status;
+
+    /** Refund status detail. */
+    private String statusDetail;
+
+    private OrderItem items;
```

Comment:
[req] Só faltou um comentário aqui.
```suggestion
    
    /** Refund items. */
    private OrderItem items;
```